SPECIFICATION.md

```diff
@@ -176,10 +176,17 @@ Je distinkter `creation_iri` werden folgende Tripel erzeugt:
 GRAPH {graph_iri} {
   <{creation_iri}>
     a crm:E65_Creation ;
-    rdf:label "{data_provider} / {file_namespace} / {media_type}" ;
+    m4p0:hasCreationPhase <https://www.museum4punkt0.de/catalogue/ontology/MaterialProduction> ;
+    m4p0:hasCreationMethod <{creation_method_concept_iri}> .
```

Comment:
```suggestion
    m4p0:hasCreationMethod <https://www.museum4punkt0.de/catalogue/ontology/Digitisation> .
    <https://www.museum4punkt0.de/catalogue/ontology/Digitization> a skos:Concept ;
    skos:inScheme  <https://www.museum4punkt0.de/catalogue/ontology/CreationMethod> ;
    skos:prefLabel "Digitalisierung"@de .
```
wäre es es so möglich? ich kann es nämlich NICHT in der app fest verankern. auch jetzt muss bei jedem löschen des storages (docker-compose down --volumes) meine vordefinierten vokabulare (die haben eben "feste" iris ohne UUID) erneut über den sparql endpoint speichern.

Reply:
1) diese suggestions sind eigentlich für andere user da. wenn du das benutzt, um deine vorigen commits zu ergänzen, dann kannst du auch deinen eigenen vorschlag gleich committen. so ist es unklar, ob ich jetzt eine beurteilung vornehmen soll.

2a) wie kommt denn die ontologie in's system? auch über den sparql-editor? ich will darauf hinaus, dass derselbe mechanismus doch auch andere tripel anlegen können sollte.

2b) ansonsten würde ich sagen, dass ein in eine datei serialisiertes basic knowledge halt einmalig zur initialisierung über das Data Import interface eingespielt werden kann. analog zum anlegen der editor-rolle.

3) 

> deswegen hätte ich gern die angabe von "hasCreationPhase" und "hasCreationMethod" direkt gespeichert.

das stellte ich nicht in frage. ich stelle das redundante anlegen des objektes zu `hasCreationMethod` in frage.